### PR TITLE
Feature/spectron-modify-action2

### PR DESCRIPTION
**Pull request by Jess-Jian**

### CODE REVIEW

File: test/features/editor/modify-action-test/group-test.js [hunk 1]
```diff
@@ -1,94 +1,110 @@
...
+    const result2 = await app.client.execute(() => {
+      const groupVisible = svgCanvas.getVisibleElements();
+      const groupLength = $('#svg_5').children().length;
+      return { groupLength, groupVisible };
+    });
+    expect(result2.groupLength).toEqual(3);
+    expect(result2.groupVisible.length).toEqual(1);
+  });
+
+  test('Check Unroup', async function () {
```
Comment (dkdkwizard) [inline test/features/editor/modify-action-test/group-test.js:43]: Check Ungroup

File: test/features/editor/modify-action-test/group-test.js [hunk 2]
```diff
@@ -1,94 +1,110 @@
...
+      let childNodes = Array.from(g.childNodes);
+      const rectangle = document.getElementById('svg_1');
+      const ellipse = document.getElementById('svg_2');
+      const polygon = document.getElementById('svg_2');
+      const isRectInsideGroup = childNodes.includes(rectangle);
+      const isEllipseInsideGroup = childNodes.includes(ellipse);
+      const isPolygonInsideGroup = childNodes.includes(polygon);
+      return { isRectInsideGroup, isEllipseInsideGroup, isPolygonInsideGroup };
+    });
+    expect(result.isRectInsideGroup).toBe(true);
```
Comment (dkdkwizard) [inline test/features/editor/modify-action-test/group-test.js:26]: `.toBeTruthy()`

File: test/features/editor/modify-action-test/group-test.js (hunk 1 above)
Comment (Jess-Jian) [reply test/features/editor/modify-action-test/group-test.js]: Updated

File: test/features/editor/modify-action-test/group-test.js (hunk 2 above)
Comment (Jess-Jian) [reply test/features/editor/modify-action-test/group-test.js]: Updated

**Review by dkdkwizard — APPROVED** [review]

**Pull request merged**